packages/gatsby-theme-vtex/src/sdk/buyButton/useBuyButton.ts

```diff
@@ -17,7 +17,15 @@ export interface SKU {
   sellers: Seller[]
 }
 
-export const useBuyButton = (sku: Maybe<SKU>, quantity: number) => {
+type Options = {
+  isOneClickBuy?: boolean
+}
+
+export const useBuyButton = (
+  sku: Maybe<SKU>,
+  quantity: number,
+  options?: Options
```

Comment:
Btw, in the future there will be a `seller` parameter

Reply:
Maybe have

`sku, quantity, seller, options`

Reply:
I agree. We should have a single options parameters